contracts/test/MockERC721TokenVault.sol

```diff
@@ -0,0 +1,26 @@
+//SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import "../external/fractional/ERC721TokenVault.sol";
+
+contract MockTokenVault is TokenVault {
```

Comment:
you shouldn't use a mock token vault for testing, this is not how it actually works in production :/ it obscures the fact that these contracts won't ultimately work, until there's an update to Fractional